doc-content/enterprise-only/integration/dev-studio-plug-in-install-proc.adoc

```diff
@@ -1,24 +1,25 @@
 
-//Get the latest Red Hat JBoss Developer Studio from the https://access.redhat.com[Red Hat Customer //Portal]. The {PRODUCT} plug-ins for Red Hat JBoss Developer Studio are available using the update site.
+//Get the latest Red Hat Developer Studio from the https://access.redhat.com[Red Hat Customer //Portal]. The {PRODUCT} plug-ins for Red Hat Developer Studio are available using the update site.
 
 .Procedure
-. Download the {PRODUCT} plug-ins for Red Hat JBoss Developer Studio:
+. Download the {PRODUCT} plug-ins for Red Hat Developer Studio:
 .. Log in to the https://access.redhat.com[Red Hat Customer Portal].
 .. Click *DOWNLOADS* at the top of the page.
-.. On the *Product Downloads* page that opens, navigate to the JBOSS DEVELOPMENT AND MANAGEMENT section, and click *Red Hat JBoss Developer Studio*.
-.. On the *Software Downloads* page, download *Red Hat JBoss Developer Studio Integration Stack 11.3.0 Update Site Zip* (`devstudio-integration-stack-11.3.0.GA-updatesite.zip`).
-//. Unzip the `devstudio-11.3.0.GA-updatesite-core.zip` file.
-. Start Red Hat JBoss Developer Studio by selecting *Red Hat JBoss Developer Studio* from the Start menu.
+.. On the *Product Downloads* page that opens, navigate to the JBOSS DEVELOPMENT AND MANAGEMENT section, and click *Red Hat Developer Studio*.
+.. On the *Software Downloads* page, from the *Version* list, select *12.0.0*, and then download *Red Hat Developer Studio Integration Stack 12.0.0 Update Site Zip* (`devstudio-integration-stack-12.0.0.GA-updatesite.zip`).
+//. Unzip the `devstudio-integration-stack-12.0.0.GA-updatesite.zip` file.
+. Start Red Hat Developer Studio by selecting *Red Hat Developer Studio* from the Start menu.
 . Click *Help* -> *Install New Software*.
-. Click *Add* to display the *Add Repository* box.
-. Enter a name in the *Name* field, for example, `devstudio-11.3.0.GA`.
-. Click *Archive*, select the `devstudio-11.3.0.GA-updatesite-core.zip` file, and click *OK*.
+. Click *Add* to display the *Add Repository* window.
+. Enter a name in the *Name* field, for example, `devstudio-12.0.0.GA`.
+. Click *Archive*, navigate to the `devstudio-integration-stack-12.0.0.GA-updatesite.zip` file, select the file, and then click *OK*.
+. Click *Add*.
 //+
-//`https://devstudio.jboss.com/11/stable/updates/integration-stack`
-. In the Install window, select *JBoss Business Process and Rule Development* from the list, click *Next*, and then click *Next* again.
-. Read the license agreement, accept it, and click *Finish*.
-. Restart Red Hat JBoss Developer Studio after the installation process finishes.
+//`https://devstudio.jboss.com/12/stable/updates/integration-stack`
+. In the Install window, select the *JBoss Business Process and Rule Development* check box from the Available Software list, click *Next*, and then click *Next* again.
```

Comment:
Bold Install:
In the Install window...
I would loose the last 'then'  - simply , and click Next again.

Reword:
...select *JBoss Business Process and Rule Development* from the Available Software list, click *Next*, and  click *Next* again.

Also, bold Available Software.

Reply:
Confirmed via private message, bold ui elements including window names.